src/deadline/client/api/_submit_job_bundle.py

```diff
@@ -210,13 +213,42 @@ def create_job_from_job_bundle(
     # Hash and upload job attachments if there are any
     if asset_references and "jobAttachmentSettings" in queue:
         # Extend input_filenames with all the files in the input_directories
+        missing_directories: set[str] = set()
         for directory in asset_references.input_directories:
+            if not os.path.isdir(directory):
+                if require_paths_exist:
+                    missing_directories.add(directory)
+                else:
+                    logger.info(
+                        f"Input path '{directory}' does not exist. Adding to referenced paths."
+                    )
```

Comment:
Thinking it might be a warning instead of info.

Reply:
Sure, I've made the change!